packages/web/src/components/comment-settings-modal/CommentSettingsModal.module.css

```diff
@@ -0,0 +1,27 @@
+.modalBody {
```

Comment:
We want to move away from CSS modules, can we move these all into inline style props throughout the JSX? It should be pretty easy but let me know if you need any help doing it!